Mail/Views/Thread/Message/MessageBannerHeaderView.swift

```diff
@@ -141,6 +145,11 @@ struct MessageBannerHeaderView: View {
             snackbarPresenter.show(message: MailResourcesStrings.Localizable.snackbarAcknowledgementFailure)
         }
     }
+
+    private func addMatomo(for banner: MessageBanner) {
```

Comment:
Can we use a name like `trackBannerEvent`? It would be more meaningful, `addMatomo` feels like we inject something